homeassistant/components/ps4/strings.json

```
@@ -19,21 +19,21 @@
         "data": {
           "region": "Region",
           "name": "[%key:common::config_flow::data::name%]",
-          "code": "PIN",
+          "code": "[%key:common::config_flow::data::pin%]",
           "ip_address": "[%key:common::config_flow::data::ip%]"
         }
       }
     },
     "error": {
       "credential_timeout": "Credential service timed out. Press submit to restart.",
-      "not_ready": "PlayStation 4 is not on or connected to network.",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "login_failed": "Failed to pair to PlayStation 4. Verify PIN is correct.",
       "no_ipaddress": "Enter the IP Address of the PlayStation 4 you would like to configure."
     },
     "abort": {
       "credential_error": "Error fetching credentials.",
-      "no_devices_found": "No PlayStation 4 devices found on the network.",
-      "devices_configured": "All devices found are already configured.",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]",
+      "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]",
```

Please change the key here
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```